src/jit/instr.h

```diff
@@ -86,13 +86,32 @@ enum GCtype : unsigned
     GCT_BYREF
 };
 
-// TODO-Cleanup:  Move 'insFlags' under _TARGET_ARM_
+#if defined(_TARGET_XARCH_)
+enum insFlags: uint8_t
+{
+    INS_FLAGS_None = 0x00,
+
```

Comment:
IMO, all the empty lines here are not necessary.

Reply:
Fixed.